bundle/manifests/odf-operator.clusterserviceversion.yaml

```diff
@@ -37,6 +37,7 @@ metadata:
     containerImage: quay.io/ocs-dev/odf-operator:latest
     description: OpenShift Data Foundation provides a common control plane for storage
       solutions on OpenShift Container Platform.
+    olm.properties: '[{"type": "olm.maxOpenShiftVersion", "value": "4.14"}]'
```

Comment:
```suggestion
    olm.properties: '[{"type": "olm.maxOpenShiftVersion", "value": "4.14.0"}]'
```

Reply:
Is this how the spec is ? Seems weird.

Reply:
Yes it is, Pls refer to the link in the description box. Also it should be `4.14` not `4.14.0`